src/pages/en/guides/configuring-astro.mdx

```diff
@@ -152,8 +152,10 @@ You can use `process.env` in a configuration file to access other environment va
 You can also use [Vite's `loadEnv` helper](https://main.vitejs.dev/config/#using-environment-variables-in-config) to manually load `.env` files.
 
 ```astro title="astro.config.mjs"
+---
 import { loadEnv } from "vite";
 const { SECRET_PASSWORD } = loadEnv(import.meta.env.MODE, process.cwd(), "");
+---
```

Comment:
It would make more sense to make it a .js file to have syntax highlighting
I'll make a PR if you don't mind @ElianCodes 🙂

Reply:
Oh yeah, actually that makes more sense 🤦